backend/src/redis/RedisClient.py

```diff
@@ -40,10 +42,25 @@ def get_and_delete(self, key):
 
         return get
 
+    # returns the key of the item in the queue
+    def add_to_q(self, json):
+        try:
+            redis_key = RedisClient.q_size() + 1
+            RedisClient.put_item(redis_key, json)
```

Comment:
I was thinking the json could be in the form of a dictionary, and this call could handle the json.dumps 
```suggestion
            RedisClient.put_item(redis_key, json.dumps(jsonObject))
```

Also renaming json --> jsonObject so theres no name conflict